.gitignore

```diff
@@ -1 +1,3 @@
 vendor/
+.idea/
+.DS_Store
```

Comment:
This two line should not be in the projet `.gitignore`, but in a global one: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#create-a-global-gitignore

Reply:
Thanks for the link, don't jnow a global gitignore can be created.